Simplify Bolt Foundry package README for better developer onboarding (#937)
bolt-foundry/bolt-foundry: docs/library-vision.md

# Bolt Foundry library vision

## What is Bolt Foundry?

Bolt Foundry is a TypeScript/JavaScript library that makes LLM applications
reliable. Instead of writing brittle prompt strings, you compose structured,
testable prompts using cards.

```typescript
// Before: Brittle, untestable prompt strings
const prompt = "You are a helpful assistant. Be concise. User: " + userInput;

// After: Structured, testable, maintainable cards
import { createCard } from "@bolt-foundry/cards";

const assistant = createCard("assistant")
  .spec("You are a helpful assistant")
  .spec("Be concise", {
    samples: [
      { text: "The answer is 42", score: 3 },
      { text: "Well, to understand this we need to go back to...", score: -3 },
    ],
  })
  .context({ userInput: "string" });
```

## Why we're building this

Every developer building LLM applications faces the same problems:

1. **Can't debug prompts**: When something goes wrong, there's no way to figure
   out why without extensive reproduction attempts
2. **Can't predict changes**: It's impossible to intuit or prove changing a few
   words won't break other situations

> 3. **Slow iteration cycles**: Testing is largely manual, and challenging (if
>    possible at all) to automate

Reviewer: nit
```suggestion
3. **Slow iteration cycles**: Testing is largely manual, and challenging (or
   impossible) to automate
```
